src/styles/components/_product-information.scss

```diff
@@ -1,20 +1,122 @@
 .product-information {
-  display: flex;
-  flex-direction: column;
-  align-items: center;
   box-sizing: border-box;
-  padding: rem(16px);
+  padding: rem(18px 24px);
+
+  @include respond-above('md') {
+    padding: 0;
+  }
+
+  &__header,
+  &__body {
+    @include respond-above('md') {
+      background-color: $alabaster;
+    }
+  }
 
   &__header {
-    margin-bottom: rem(24px);
+    display: flex;
+    align-items: center;
+
+    @include respond-above('md') {
+      width: rem(200px);
+      padding: rem(20px) 0 0 rem(28px);
+      border-top-left-radius: rem(32px);
+    }
+
+    @include respond-above('lg') {
+      width: rem(220px);
+      padding: rem(24px) 0 0 rem(32px);
+    }
+  }
+
+  &__subheader {
+    display: none;
+
+    @include respond-above('md') {
+      display: block;
+      margin-bottom: rem(20px);
+    }
+
+    @include respond-above('lg') {
+      margin-bottom: rem(24px);
+    }
   }
 
   &__title {
-    margin: 0 0 rem(16px) 0;
+    all: unset;
+
+    font-size: rem(22px);
+    line-height: 1.48;
+
+    @include respond-above('sm') {
+      font-size: rem(24px);
+    }
+
+    @include respond-above('md') {
+      font-size: rem(16px);
+      font-weight: 600;
+    }
+
+    @include respond-above('lg') {
+      font-size: rem(19px);
+    }
+  }
+
+  &__subtitle {
+    all: unset;
+
+    color: $cat-black;
+    font-size: rem(19px);
+    font-weight: 500;
+    line-height: 1.5;
+
+    @include respond-above('lg') {
+      font-size: rem(22px);
+    }
   }
 
   &__description {
     margin: 0;
-    text-align: center;
+
+    color: $iron;
+    font-size: rem(14px);
+    line-height: 1.3;
+
+    @include respond-above('sm') {
+      font-size: rem(16px);
+    }
+
+    @include respond-above('md') {
+      color: $darker-gunpowder-gray;
+      font-size: rem(17px);
```

Comment:
Then it will be the same size as title (`Product Details`). As another option, do you think we should make title bigger than description? See below.

Same size
<img width="630" alt="Screen Shot 2564-03-27 at 23 48 09" src="https://user-images.githubusercontent.com/32285706/112727872-3049c600-8f57-11eb-9f14-1fad748b01d1.png">
Title is bigger
<img width="569" alt="Screen Shot 2564-03-27 at 23 47 47" src="https://user-images.githubusercontent.com/32285706/112727877-33dd4d00-8f57-11eb-806f-5f410377ccc3.png">

Reply:
I cheer the 2nd one.